framework/doc/content/documentation/systems/InterfaceKernels/index.md

```diff
@@ -1,11 +1,28 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
+# InterfaceKernels System
 
+Interface kernels are meant to assist in coupling different physics across sub-domains. The most straightforward example is the case in which one wants to set the flux of a specie A in subdomain 0 equal to the flux of a specie B in subdomain 1 at the boundary between subdomains 0 and 1. In mathematical terms, we might be interested in establishing the condition:
 
-# InterfaceKernels System
+$$$-D_0 \frac{\partial c_0}{\partial x} = -D_1 \frac{\partial c_1}{\partial x}$$$
+
+where $$$D_i$$$ is the diffusion coefficient of specie $$$i$$$ in subdomain $$$i$$$, and $$$c_i$$$ is the concentration of specie $$$i$$$ in subdomain $$$i$$$. An example of this condition is shown in the Moose test directory; see files below:
+
+[coupled_value_coupled_flux.i](https://github.com/idaholab/moose/blob/devel/test/tests/interfacekernels/1d_interface/coupled_value_coupled_flux.i)
```

Comment:
@aeslaughter Is this how we link files in the MooseDocs? I thought there was a syntax for this...